src/topp/IsobaricWorkflow.cpp

```diff
@@ -483,6 +520,21 @@ class TOPPIsobaricWorkflow :
       vector<PeptideIdentification> pep_ids;
       FileHandler().loadIdentifications(id_file, prot_ids, pep_ids);
       // TODO filter by qvalue here?
+      double pro_score = getDoubleOption_("protein_score");
+      double pep_score = getDoubleOption_("peptide_score");
+
+      if (!std::isnan(pro_score)) 
+      {
+        OPENMS_LOG_INFO << "Filtering by protein score (better than " << pro_score << ")..." << endl;
+        IDFilter::filterHitsByScore(prot_ids, pro_score);
+      }
+
+      if (!std::isnan(pep_score)) 
+      {
+        OPENMS_LOG_INFO << "Filtering by peptide score (better than " << pep_score << ")..." << endl;
+        IDFilter::filterHitsByScore(pep_ids, pep_score);
```

Comment:
I think this filters by PSM score not by peptide score.

Reply:
Yes, depends on how FDR was performed and which PSMs are left (if only one PSM per peptide is left, due to aggregation or max filtering, it is equivalent with peptide FDR). PSM FDR is definitely the default, if you dont do special stuff.

Reply:
what is the case here in IsobaricWorklow? I think PSM?